### DISCUSSION TITLE
Funnier UI

### DISCUSSION THREAD

**Discussion by Ishfaq24** [aqib-m31/Jokesaurus-The-Hilarious-Dictionary#20] [Ideas]
Can we add an animated emoji to it? 
Like it should appear each time the search box is pressed!🤔

**Comment by aqib-m31** [comment]
This is a great idea. 😀
We'll surely try adding this. Could you please create an issue for the same.

**Reply by aqib-m31** [reply]
What say @unnatijp24?

**Reply by Ishfaq24** [reply]
I created a Feature Request issue #21
